3_javascript/javascript1/week1/homework/dogage.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>How old will be dog in the future</title>
+  </head>
+  <body>
+    <script>
+      const shouldShowResultInDogYears = true;
+      const dogYearOfBirth = 2017;
+      const dogYearFuture = 2027;
+      const dogYear = dogYearFuture - dogYearOfBirth;
+
+      if (shouldShowResultInDogYears) {
+        console.log(
+          "Your dog will be" + dogYear * 15 + "human years old in 2027"
```

Comment:
Here you need a space between the variable and text
```suggestion
          "Your dog will be " + dogYear * 15 + " human years old in 2027"
```
or it will be even better if you use template literals instead. [Here](https://www.w3schools.com/js/js_string_templates.asp) you can read more about it. It makes code more readable and elegant :wink:
```suggestion
          `Your dog will be ${dogYear * 15} human years old in 2027`
```